### DISCUSSION TITLE
Mod Wii U to get it working

### DISCUSSION THREAD

**Discussion by ricky074game** [vanilla-wiiu/vanilla#65] [Ideas]
Hello,
Looking through the Wi-Fi card complication and all the troubles the Wii U does in order to connect with a game pad, is it possible to reverse it. So, basically mod the Wii U so that it can connect to the game pad using standard Wi-Fi practices. 
If that is possible, you could split the project into 2, this version which allows connecting to an unmolded Wii U to get it modded, and then a modified version to connect normally and maybe support all the network adapters, and maybe support iOS and android too?

Tell me what you think about this.

**Comment by spicyegg234** [comment]
https://github.com/vanilla-wiiu/vanilla/discussions/299 (technically 299 is a duplicate of #65 (this one) but this should give you your answer)
